harvard-lts/etd_alma_service
# Added healthchecks for alma

**Added healthchecks for alma**
* * *

**JIRA Ticket**: https://jira.huit.harvard.edu/browse/ETD-190

# What does this Pull Request do?
adds alma hc

# How should this be tested?

this has been deployed to dev.  to test,
1) run the integration test: ltsds-cloud-dev-1.lib.harvard.edu:10610/integration
2) find the container in the cloud
3) go to the node containing that container
4) `docker inspect CONTAINER_ID | grep healthy` should show the HC working

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n (but itest should work)

# Interested parties
Tag (@ mention) interested parties: @awoods
